autowarefoundation/autoware
# Feature/incremental ndt cpu

## Status
**DEVELOPMENT**

## Description
Update the nearest neighbor search using octree. This search is used in getFitnessScore function.
## Related PRs
## Todos
- [x ] Tests


## Steps to Test or Reproduce
1. Select "pcl_anh" and check  "incremental voxel update" from the app button of ndt_mapping.

1. launch ndt_mapping.

Be careful as it still consumes a lot of memory when build a large map.
